### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by asafshen**
add readme
remove tokens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# java-validate-sample
\ No newline at end of file
+# java-validate-sample
+A simple package for Descope JWT validation for Java, along with a test
+
+## Prerequisites 
+- Install [Java](https://www.java.com/en/download)
```
Comment (slavikm) [inline README.md]: Also maven as dev dependency, I guess

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+## Quick start
+
+```java
+var PROJECT_ID = "<Descope-Project-ID>";
+var TEST_JWT = "<JWT>";
+var jwtValidation = new JwtValidation(PROJECT_ID);
+try {
+    var t = jwtValidation.validateAndCreateToken(TEST_JWT);
+    System.out.printf("JWT token %s%n", t);
```
Comment (slavikm) [inline README.md]: Does this work? `%n` vs `\n`?

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+} catch (Exception e) {
+    // print e
+    System.out.printf("JWT validation error %s\n", e.getMessage());
+}
+```
+ 
+## Run test
+See src/test folder.
```
Comment (slavikm) [inline README.md]: Maven has test lifecycle method: `mvn test`

File: README.md (hunk 1 above)
Comment (asafshen) [reply README.md]: added 👍

**Review by slavikm — APPROVED** [review]

**Pull request merged**